packages/react-strict-dom/tests/__snapshots__/html-test.js.snap-native

```diff
@@ -118,31 +83,10 @@ exports[`html "a" supports global attributes 1`] = `
 
 exports[`html "a" supports inline event handlers 1`] = `
 <Text
-  onAuxClick={[Function]}
-  onBlur={[Function]}
-  onChange={[Function]}
-  onClick={[Function]}
-  onContextMenu={[Function]}
-  onCopy={[Function]}
-  onCut={[Function]}
-  onFocus={[Function]}
-  onFocusIn={[Function]}
-  onFocusOut={[Function]}
-  onFullscreenChange={[Function]}
-  onFullscreenError={[Function]}
   onGotPointerCapture={[Function]}
-  onInput={[Function]}
-  onKeyDown={[Function]}
-  onKeyUp={[Function]}
   onLostPointerCapture={[Function]}
-  onMouseDown={[Function]}
```

Comment:
Probably need to keep the legacy mouse events around a bit longer for old internal web code